City-of-Helsinki/helsinki-paatokset
# PP-299: Added anchor links policymaker listing and links to decision tree

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-299

Added id:s to accordions on policymaker search page and added links to decision tree

**How to test**

Run `drush cr` and go to https://helsinki-paatokset.docker.so/fi/tietoa-paatoksenteosta

Check that the color boxes are now links and they take the user to the correct places. 

(Mobile fixes are in another ticket)
